setup.py

```diff
@@ -649,7 +680,8 @@ def setup_package():
         ext_modules = cythonize(
             ext_modules, compile_time_env=compile_env,
             include_path=list(include_path),
-            language="c++"
+            language="c++",
+            compiler_directives=COMPILER_DIRECTIVES,
```

Comment:
`cythonize` is only called for Cython >= 0.25, will the tracing still work in those cases?